AirCasting/CoreData/Session.swift

```diff
@@ -118,7 +122,12 @@ public enum SessionStatus: Int {
     case NEW = -1
     case RECORDING = 0
     case FINISHED = 1
-    case DISCONNETCED = 2
+    case DISCONNECTED = 2
+}
+
+enum SessionFollowing: Int {
```

Comment:
Suggestion mode! Maybe change cases to "followed" & "unfollowed"? :)

Reply:
I will allow myself to leave it as it is now, because I think that fixed is entry state. And then you can additionally make follow state, but when we go back to entry state (fixed) I think there is no need to say it is unfollow or etc. If it's ok I will leave it 😄

Reply:
Sure, no need to explain, it was just a suggestion :)